src/frontend/components/SøknadsSteg/Kvittering/Kvittering.tsx

```diff
@@ -49,30 +57,39 @@ const Kvittering: React.FC = () => {
         }
     }, []);
 
+    const kvitteringTekster = tekster().KVITTERING;
+
     return (
         <Steg tittel={<TekstBlock block={kvitteringTekster.kvitteringTittel} />}>
-            <KomponentGruppe>
+            <VStack gap="12">
                 <Alert variant="success">
                     <TekstBlock
                         block={kvitteringTekster.soeknadMottatt}
                         flettefelter={{ dato, klokkeslett }}
                     />
                 </Alert>
-            </KomponentGruppe>
-            <KomponentGruppe>
-                <TekstBlock
-                    block={kvitteringTekster.infoTilSoker}
-                    typografi={Typografi.BodyShort}
-                />
-            </KomponentGruppe>
 
-            {varEøsSøknad && (
-                <KomponentGruppe>
-                    <TekstBlock block={kvitteringTekster.kontonummerEOES} />
-                </KomponentGruppe>
-            )}
```

Comment:
Skal ikke denne informasjonen om kontonummer for EØS-søknader vises lenger?

Reply:
Nei, innholdet i den nye kontoinformasjon komponenten inneholder all informasjon om kontonummer, så den gamle kontonummerEOES teksten er ikke lengre relevant.